centreon/src/Core/Security/Vault/Domain/Model/NewVaultConfiguration.php

```diff
@@ -34,9 +34,11 @@ class NewVaultConfiguration
 {
     public const MIN_LENGTH = 1;
     public const MAX_LENGTH = 255;
+    public const NAME_MAX_LENGTH = 50;
     public const MIN_PORT_VALUE = 1;
     public const MAX_PORT_VALUE = 65535;
     public const SALT_LENGTH = 128;
+    public const NAME_VALIDATION_REGEX = '/^[\w+\-_\/]*$/';
```

Comment:
`\w` iq equivalent o `[A-Za-z0-9_]`, then please remove the superfluous `_`. 

```suggestion
    public const NAME_VALIDATION_REGEX = '/^[\w+\-\/]*$/';
```